tests/integration.rs

```diff
@@ -1463,3 +1397,573 @@ route = "/..."
         Ok(())
     }
 }
+
+// TODO(Reviewer): How can I move this to a new file? I wasn't able to get the imports to work out.
```

Comment:
I couldn't figure out how to put this module in another file, but still import things from the testcases module.